src/app/onboarding/views/MultiZoneView.vue

```diff
@@ -120,6 +122,8 @@ async function getZoneIngresses() {
 
     hasZoneIngresses.value = total > 0
   } catch (error) {
+    hasZoneIngresses.value = false
+
```

Comment:
Just out of interest, were these additions required to fix the flake? (btw happy for these changes to be here it's just out of interest).

Also I spotted a typo/spelling mistake here with `LONG_POOLING_INTERVAL` > `LONG_POLLING_INTERVAL`, might as well fix that up whilst you are here? Up to you though. Lmk and I can approve either way.

Reply:
> Just out of interest, were these additions required to fix the flake? (btw happy for these changes to be here it's just out of interest).

No, they didn’t affect the outcome.